vcbuild.bat

```diff
@@ -58,9 +59,10 @@ if /i "%1"=="nosnapshot"    set nosnapshot=1&goto arg-ok
 if /i "%1"=="noetw"         set noetw=1&goto arg-ok
 if /i "%1"=="noperfctr"     set noperfctr=1&goto arg-ok
 if /i "%1"=="licensertf"    set licensertf=1&goto arg-ok
-if /i "%1"=="test"          set test_args=%test_args% abort doctool known_issues message parallel sequential addons -J&set lint_cpp=1&set lint_js=1&set build_addons=1&goto arg-ok
-if /i "%1"=="test-ci"       set test_args=%test_args% %test_ci_args% -p tap --logfile test.tap abort doctool inspector known_issues message sequential parallel addons&set cctest_args=%cctest_args% --gtest_output=tap:cctest.tap&set build_addons=1&goto arg-ok
+if /i "%1"=="test"          set test_args=%test_args% abort doctool known_issues message parallel sequential addons addons-napi -J&set lint_cpp=1&set lint_js=1&set build_addons=1&goto arg-ok
```

Comment:
> Looks like all the `napi` tests are failing on windows
> 
> an example
> 
> ```
> not ok 330 addons-napi/2_function_arguments/test
>   ---
>   duration_ms: 0.254
>   severity: fail
>   stack: |-
>     module.js:478
>         throw err;
>         ^
>     
>     Error: Cannot find module './build/Release/binding'
> ```

This line should `set build_addons_napi=1` so that the bindings for the n-api tests are compiled.